packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -101,6 +113,8 @@ export function DatePicker({
         disabled={disabled}
         readOnly={readonly}
         onChange={handleChange}
+        maxDate={maxDate}
```

Comment:
I know this is just a range but do you need a simple check for minDate < maxDate ?
#nonBlockingComment

Reply:
I thought about this as well but what should be the behavior when the developer puts maxDate < minDate? What's your suggestion? 

This is what happens now

![image](https://user-images.githubusercontent.com/31483287/154537405-cc41db4b-1278-4010-9269-037ac105c21f.png)

Reply:
Since it disables the entire range if by mistake we interchange min and max, I think a simple test should suffice

Reply:
But what I would be testing for? This behavior is exactly the same one for react-datepicker

![image](https://user-images.githubusercontent.com/31483287/154544030-ee55378a-6f0b-4de3-a462-1d86d6d589f2.png)

A test for "minDate shouldn't be grater than maxDate" makes sense to me if we had some visual feedback or let's say if we tested the case of all dates being disable (this I wouldn't know how to implement) like querying for all the dates and checking if --disabled class is present? I'm really not sure what to implement 😕

I think what react-datepicker decided was to leave for the developer the responsibility to make sure the dates are correct. Do you think there will be a problem if we did that as well? or do you have a test scenario and some tips on how to implement it?

Reply:
Also Can do something like this
```suggestion
         maxDate={maxDate && minDate && maxDate > minDate ? maxDate : undefined}
        minDate={maxDate && minDate && maxDate < minDate ? minDate : undefined}
```

Reply:
maxDate={maxDate && minDate && maxDate > minDate ? maxDate : undefined}, if minDate is not provided which is a valid case, maxDate the statement would return undefined right? The same case for minDate.

Reply:
I think you are right,  it makes sense to leave it to the user to select the correct range, 
my only concern is if a genuine mistake happens, we should fail gracefully.
(if we interchange the min and max by mistake it should not disable the calendar)

> But what I would be testing for? This behavior is exactly the same one for react-datepicker
> 
> ![image](https://user-images.githubusercontent.com/31483287/154544030-ee55378a-6f0b-4de3-a462-1d86d6d589f2.png)
> 
> A test for "minDate shouldn't be grater than maxDate" makes sense to me if we had some visual feedback or let's say if we tested the case of all dates being disable (this I wouldn't know how to implement) like querying for all the dates and checking if --disabled class is present? I'm really not sure what to implement 😕
> 
> I think what react-datepicker decided was to leave for the developer the responsibility to make sure the dates are correct. Do you think there will be a problem if we did that as well? or do you have a test scenario and some tips on how to implement it?

True, we don't have any such feedback also we are mocking dates, so that doesn't help either. 
I can't think of any better way to write a test what can we do though is fail gracefully if the dates are not in order. Regardless, this is by no means is a blocking comment.

Reply:
I see what you mean, I think this discussion is valuable. What do you meant by failing gracefully in this scenario? Unfortunately i believe that disabling any date makes since because the Union of both sets is all the dates, so i really don't know what would be a good approach on this, we could move this forward and start a discussion about this, have move people engaged as well...